### PR TITLE
DOCS-5026 Object Store FAQ limit topic

### PR DESCRIPTION
**Pull request by george-hoffer**
https://www.mulesoft.org/jira/browse/DOCS-5026

### CODE REVIEW

File: modules/ROOT/pages/osv2-faq.adoc [hunk 1]
```diff
@@ -25,6 +25,14 @@ Mule 3.8.5 and above, and all 4.x versions.
 
 ====
 
+== What happens when the transaction limit is reached?
```
Comment (steveCluster) [inline modules/ROOT/pages/osv2-faq.adoc]: What happens when the transaction limit for my subscription is reached?

Comment (george-hoffer) [reply modules/ROOT/pages/osv2-faq.adoc]: Fixed as requested

File: modules/ROOT/pages/osv2-faq.adoc [hunk 2]
```diff
@@ -25,6 +25,14 @@ Mule 3.8.5 and above, and all 4.x versions.
...
+
+Object Store v2 allows up to 100 million transactions for Object Store v2 for an Anypoint Platform organization. This limit is applied to both Production and non-Production environments.
```
Comment (steveCluster) [inline modules/ROOT/pages/osv2-faq.adoc]: One Object Store v2 subscription pack allows up to 100 million transactions per month for an Anypoint Platform organization.  This limit is applied across all Production and non-Production environments.

Comment (george-hoffer) [reply modules/ROOT/pages/osv2-faq.adoc]: Fixed as requested except that I changed "One" to "An"

**Review by steveCluster — APPROVED** [review]

**Review by wwarner-salesforce — APPROVED** [review]
Looks great!

**Pull request merged**